source/2020-02-12-ember-3-16-released.md

```diff
@@ -22,6 +24,10 @@ Ember.js is the core framework for building ambitious web applications.
 
 ### Changes in Ember.js 3.16
 
+The 3.16.0 release is an Ember.js Long-Term Support candidate. In six weeks, the 3.16.x series will become the latest LTS release and six weeks after that the 3.8 LTS branch will no longer receive bugfix patches. LTS versions allow teams to upgrade less frequently while still getting support from the Ember project and the wider ecosystem.
```

Comment:
I wasn't sure if writing "six weeks after that the 3.8 LTS branch will no longer receive bugfix patches" is correct.

The 3.12 release note showed 3.4 in this paragraph, so I assumed the difference should be 0.8. However, the 3.8 release note also showed 3.4.

Reply:
```suggestion
The 3.16.0 release is an Ember.js Long-Term Support candidate. In six weeks, the 3.16.x series will become the latest LTS release and six weeks after that the 3.12 LTS branch will no longer receive bugfix patches. LTS versions allow teams to upgrade less frequently while still getting support from the Ember project and the wider ecosystem.
```

That must have been a mistake. It should be `3.12` in this case.

Reply:
Ah, ok.